parlai/scripts/train_model.py

```diff
@@ -445,6 +431,9 @@ def save_model(self, suffix=None):
             except KeyboardInterrupt:
                 pass
 
+    def _safe_report(self, report):
```

Comment:
This might be more generally useful as a helper function beyond train_model.py, I wonder